anydex/wallet/ethereum/eth_provider.py

```diff
@@ -158,6 +158,38 @@ def send_request(self, path, data={}, method='get'):
         return response
 
     def get_balance(self, address):
+        response = self.send_request(f"/dashboards/address/{address}")
+        return response.json()["data"][address.lower()]["address"]["balance"]
+
+    def get_transaction_count(self, address):
+        # Todo: return also unconfirmed txs
+        response = self.send_request(f"/dashboards/address/{address}")
+        return response.json()["data"][address.lower()]["address"]["transaction_count"]
+
+    # def estimate_gas(self, tx):
+    #     # Todo estimate the gas better or just set to the max for smiple transactions (21000)
+    #     response = self.send_request("/stats")
+    #     return response.json()["data"]["median_simple_transaction_fee_24h"]
+
+    def get_gas_price(self):
+        response = self.send_request("/stats")
+        return response.json()["data"]["mempool_median_gas_price"]
+
+    def submit_transaction(self, raw_tx):
+        response = self.send_request("/push/transactions", data={"data": raw_tx}, method="post")
+        return response.json()["data"]["transaction_hash"]
+
+    def get_transactions_received(self, address):
+        response = self.send_request("/transactions", data={"q": f"recipient({address})"})
+        response_mempool = self.send_request("/mempool/transactions", data={"q": f"recipient({address})"})
+        txs = response.json()["data"] + response_mempool.json()["data"]
+        return self._normalize_transactions(txs)
+
+    def get_transactions(self, address):
+        sent = self.send_request("/transactions", data={"q": f"sender({address})"})
+        sent_data = sent.json()["data"]
```

Comment:
Variable seems to never be used.

Reply:
This portion of the code is from Rahim's branch when I merged master into the Monero_wallet branch locally. So I am not sure whether it should be removed.